src/amr/TiogaAMRIface.cpp

```diff
@@ -349,6 +351,12 @@ void TiogaAMRIface::update_solution()
         rnorm, amrex::ParallelDescriptor::IOProcessorNumber());
     amrex::Print() << "TIOGA interpolation error (max L2 norm) for AMR mesh: "
                    << rnorm << std::endl;
+
+    if(iproc==0){
```

Comment:
actually, can we test on if this is the IO proc?

Reply:
Made this change in a new commit